tests/upgrades/test_repository.py

```diff
@@ -203,3 +203,82 @@ def test_post_scenario_custom_repo_check(self, target_sat, pre_upgrade_data):
         )[0]
         result = rhel_client.execute(f'yum install -y {FAKE_4_CUSTOM_PACKAGE_NAME}')
         assert result.status == 0
+
+    @pytest.mark.pre_upgrade
+    def test_pre_scenario_custom_repo_check_sca_toggle(
```

Comment:
Please add a new class for the new scenario, there is no harm in keeping it under one class, but then it's easy to identify them separately as a scenario.